packages/language-server/test/plugins/typescript/features/CompletionProvider.test.ts

```diff
@@ -115,6 +115,51 @@ function test(useNewTransformation: boolean) {
             });
         });
 
+        const editingTestPositionsNewOnly: Array<[number, number, string]> = [
```

Comment:
These four doesn't work in v1. Wondering if it still makes sense to make it work.

Reply:
I wouldn't worry too much about the old transformation at this point, so I'm fine with it only working for the new transformation